web-client/public/manifest.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
I also see that the files for PWA support have been removed. Was this done on purpose?

Reply:
Done, Installed vite-plugin-pwa, and added PWA manifest in the vite.config.ts